### DISCUSSION TITLE
Add Information About How to Customize the Logo in `Reveal.js`

### DISCUSSION THREAD

**Discussion by RoyiAvital** [quarto-dev/quarto-web#274] [General]
In a presentation mode based on `revealjs` there is an option to define a logo and a footer.
I think it would be great to add a documentation on how to customize the logo:

1. Size.
2. Location.
3. Link (I want that a click on the image will link to a URL).

It would also be beneficial to remind that the `footer` field might be treated as a MarkDown text.

**Comment by mcanouil** [comment]
All (at least size and location) that are CSS based modifications.

For example: https://github.com/mcanouil/quarto-revealjs-coeos/blob/main/_extensions/coeos.scss will produce:

[Click to see the HTML<br>![](https://raw.githubusercontent.com/mcanouil/publishing-templates/main/thumbs/quarto-coeos.png)](https://m.canouil.fr/publishing-templates/quarto-coeos.html)

**Reply by RoyiAvital** [reply]
OK, That's a great starting point. 

Were can I find the default RevealJS CSS on Quarto? How can I override (Only a subset) it?

The CSS can assist with the size and location, any idea about how to adjust the link?

**Reply by mcanouil** [reply]
The documentation website has a page about styling/theming your documents: https://quarto.org/docs/output-formats/html-themes.html
Beyond that if you are looking only at HTML output formats, online resources are everywhere about HTML and CSS. Not everything are or should be on the website in my opinion.

**Reply by RoyiAvital** [reply]
I have seen those. I was looking for something on the ReavelJS mode.

**Reply by mcanouil** [reply]
Then, there is the revealjs website (https://revealjs.com/) which is comprehensive.

**Reply by RoyiAvital** [reply]
But the logo and footer are properties of Quarto, not the RevealJS. I wonder where to see the CSS Quarto uses. Where it parses the `logo` variable. Etc...

**Reply by mcanouil** [reply]
Take a look at the source code: https://github.com/quarto-dev/quarto-cli/tree/main/src/format/reveal and associated resources https://github.com/quarto-dev/quarto-cli/tree/main/src/resources/formats/revealjs

**Reply by mcanouil** [reply]
Also, you can still use JS to customise, i.e., remove the logo on title slide as described in the following issue https://github.com/quarto-dev/quarto-cli/issues/1391
